Framework/Geometry/src/Objects/Object.cpp

```diff
@@ -725,15 +725,10 @@ int Object::procString(const std::string &Line) {
   }
   // Do outside loop...
   int nullInt;
-  while (procPair(Ln, RuleList, nullInt))
-    ;
-
-  if (RuleList.size() != 1) {
-    std::cerr << "Map size not equal to 1 == " << RuleList.size() << '\n';
-    std::cerr << "Error Object::ProcString : " << Ln << '\n';
-    exit(1);
-    return 0;
+  while (procPair(Ln, RuleList, nullInt)) {
   }
+
+  assert(RuleList.size() == 1);
```

Comment:
The logic of the code should mean that RuleList is always size 1 by this point. On second thoughts I'm thinking that this should probably throw rather than use an assert. I'll update this too.